resources/public/api/conf/2.0/retrieveCodingOutUnderpayments.yaml

```diff
@@ -2,37 +2,15 @@ description: |
   This endpoint enables you to retrieve HMRC-held coding out underpayments and debt amounts or user submitted amended coding out amounts for a previous tax year and given National Insurance number.
   
   ### Test data
-  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
-  <table>
-      <thead>
-          <tr>
-              <th>Header Value (Gov-Test-Scenario)</th>
-              <th>Scenario</th>
-          </tr>
-      </thead>
-      <tbody>
-          <tr>
-              <td><p>N/A - DEFAULT</p></td>
-              <td><p>Simulates a successful response with latest values.</p></td>
-          </tr>
-          <tr>
-              <td><p>N/A - DEFAULT<br>using <strong>?source=latest</strong></p></td>
-              <td><p>Simulates a successful response with latest values.</p></td>
-          </tr>
-          <tr>
-              <td><p>N/A - DEFAULT<br>using <strong>?source=hmrcHeld</strong></p></td>
-              <td><p>Simulates a successful response with HMRC-held values.</p></td>
-          </tr>
-          <tr>
-              <td><p>N/A - DEFAULT<br>using <strong>?source=user</strong></p></td>
-              <td><p>Simulates a successful response with user submitted values.</p></td>
-          </tr>
-          <tr>
-              <td><p>NOT_FOUND</p></td>
-              <td><p>Simulates the scenario where no data is found.</p></td>
-          </tr>
-      </tbody>
-  </table>
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario)        | Scenario                                                    |
+  |-----------------------------------------|-------------------------------------------------------------|
+  | N/A - DEFAULT                           | Simulates a successful response with latest values.         |
+  | N/A - DEFAULT<br>using ?source=latest   | Simulates a successful response with latest values.         |
```

Comment:
the query parameters used to be inside `<strong><strong/>` tags. We can do this in markdown by adding `**` either side of the text

Reply:
As discussed, I have kept it not bold to be consistent with that in v1.